src/debug.h

```diff
@@ -3,8 +3,8 @@
 
 #ifdef DEBUG
 # include <stdio.h>
-# define PSTDERR(fmt, args...) do { dprintf(2,fmt, ## args); } while(0)
-# define PDEBUG(fmt, args...) PSTDERR("DEBUG:"fmt, ## args)
+# define PSTDERR(fmt, args...) do { dprintf(2,fmt, getpid(), ## args); } while(0)
```

Comment:
there's one occurence of PSTDERR() in debug.c. i suspect that one will get the arguments shuffled around unexpectedly with this change. i guess it'd be safer to put the getpid call directly into the PDEBUG macro, rather than spreading the functionality over 2 macros depending on each other

Reply:
Hah! Good catch, that was unintentional... 3am coding...